.travis.yml

```diff
@@ -6,14 +6,18 @@ rvm:
   - 2.3.4
   - 2.2.7
   - jruby
+bundler_args: ''
```

Comment:
Makes it difficult to run locally because of the `--deployment` argument. It generates a bunch of untracked files for git.